### ISSUE TITLE
Cost Profile: aggregateCost vs costType of Aggregated Costs

### ISSUE THREAD

**Issue by siuc-nate** [CredentialTransparencyInitiative/vocabularies#69]
Currently Cost Profile has an "aggregateCost" property. Would it not be better to have a costType enumeration value of "aggregatedCosts"?

**Comment by stuartasutton** [comment]
No. My understanding is that this property was deliberately added a few weeks back to hold an actual cost (i.e. $$$$$) -- note the range of xsd:float (http://www.credreg.net/page/propertiesviewer#aggregateCost). Including Credential as domain as opposed to only CostProfile might also be considered making it possible to assert an aggregate cost for a credential.

**Comment by siuc-nate** [comment]
We added CostProfile to Credential, Learning Opportunity Profile, and Assessment Profile in order to express the summarized costs for those objects - but the intent was always that the whole CostProfile be used.

CostProfile itself has a "price" property - does this not make "aggregateCost" redundant (particularly as a CostProfile is only used to describe one cost)? How would one use both price and aggregateCost together?  If we already have an enumeration of CostTypes, it seems to me that aggregate costs are just one of those types.

**Comment by stuartasutton** [comment]
Nate, I don't know what to do about this. It was added in response to a problem identified by Mike and Jeanne. I just resent to all 4 of us the original email thread.

**Comment by siuc-nate** [comment]
Perhaps it would be best left to tomorrow's meeting/call?

**Comment by siuc-nate** [comment]
After reviewing this with the team, it appears that our costType vocabulary doesn't currently include aggregateCost - but I think it would serve the purpose better if it did. The user could select a cost type of "aggregate cost" and use the existing "price" field to indicate the cost.  This also prevents a situation where a costProfile could have both an aggregate cost and a cost type of something else and a price. It also prevents the need to enter aggregate cost in every instance of cost profile used to express a range of other cost types.

**Comment by stuartasutton** [comment]
***ACTIONS TAKEN:***

1. The concept aggregateCost added to costType vocabulary in vocabulary spreadsheet with a definition of "The sum of direct costs."; and
2. The concept was added to the GitHub turtle version of the vocabulary.

**Comment by siuc-nate** [comment]
Are we keeping the aggregateCost property? It's still in the spreadsheet.

**Comment by stuartasutton** [comment]
Probably not needed. Mike and Jeanne originally asked for the property to fill a need. If "the team" includes both of them, let's remove it.

**Comment by stuartasutton** [comment]
I have reopened this issue because:

1. We still have the ctdl:aggregateCost property; and
2. The concept aggregateCost was added to CostType vocabulary in Github and in the spreadsheet by me and is now not there in the spreadsheet (sync problem); and
3. An actual AggregateCost vocabulary was added to the spreadsheet with a single concept-aggregateCost. This separate vocabulary to contain one concept that can be as easily accommodated in the existing CostType vocabulary makes no sense to me.  

Anyway, there is some cleaning up to do here.
